components/datepicker/datepicker-inner.component.ts

```diff
@@ -310,7 +310,16 @@ export class DatePickerInnerComponent implements OnInit, OnChanges {
   }
 
   private isDisabled(date:Date):boolean {
-    // todo: implement dateDisabled attribute
+    if(this.dateDisabled !==undefined && this.dateDisabled!==null){
+		for(var i=0;i<this.dateDisabled.length;i++){
+			var disabledDate = this.dateDisabled[i].getDate() + "/" +(this.dateDisabled[i].getMonth()+1) + "/" +(this.dateDisabled[i].getFullYear());
```

Comment:
If you change this.dateDisabled[i] to this.dateDisabled[i].date for each of your function calls this works perfectly.